packages/react-app/components/notifications/OnboardingBatch.jsx

```diff
@@ -13,23 +13,43 @@ import {
 } from "@chakra-ui/react";
 import { getUpdateBatchSignMessage, postUpdateBatch } from "../../data/api";
 import { useUserAddress } from "eth-hooks";
+import { SERVER_URL } from "../../constants";
+import { useState, useCallback, useEffect } from "react";
+import axios from "axios";
+import moment from "moment";
 
-const NEXT_BATCH_TELEGRAM = process.env.NEXT_PUBLIC_BATCH_TELEGRAM;
-const NEXT_BATCH_DATE = process.env.NEXT_PUBLIC_BATCH_DATE;
-const NEXT_BATCH = process.env.NEXT_PUBLIC_BATCH_NUMBER;
+const serverPath = "/batches/latest-open";
 
 const OnboardingBatch = ({ notification, onMarkAsRead, builder, userProvider, onUpdate }) => {
   const address = useUserAddress(userProvider);
+  const [latestOpenBatch, setLatestOpenBatch] = useState(null);
+  const [isLoadingOpenBatch, setIsLoadingOpenBatch] = useState(false);
 
   const hasBatchNumber = builder?.batch?.number;
-  const isInRecentBatch = builder?.batch?.number === NEXT_BATCH;
+  const isInRecentBatch = builder?.batch?.number === latestOpenBatch?.name;
 
   const toast = useToast({ position: "top", isClosable: true });
   const toastVariant = useColorModeValue("subtle", "solid");
 
+  const fetchLatestOpenBatch = useCallback(async () => {
+    setIsLoadingOpenBatch(true);
+    try {
+      const fetchedBatch = await axios.get(SERVER_URL + serverPath);
+      setLatestOpenBatch(fetchedBatch.data.data);
+    } catch (error) {
+      console.error("Error fetching batch:", error);
+    } finally {
+      setIsLoadingOpenBatch(false);
+    }
+  }, []);
+
+  useEffect(() => {
+    fetchLatestOpenBatch();
+  }, [fetchLatestOpenBatch]);
+
```

Comment:
This can be simplified a bit (no need for useCallback) if we just add the function inside the useEffect)

https://github.com/scaffold-eth/buidlguidl-v3/pull/117/commits/177b4662516216a0eae6f4f505135cbd54ef2a3b